lambda/models/state_machine/create_model.py

```diff
@@ -97,14 +99,61 @@ def handle_poll_docker_image_available(event: Dict[str, Any], context: Any) -> D
 def handle_start_create_stack(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
     """Start model infrastructure creation."""
     output_dict = deepcopy(event)
+    request = CreateModelRequest.validate(event)
+
+    prepared_event = {}
+
+    def camelize_object(o):
```

Comment:
this is a lot of work to transform an object that we control ourselves. aside from the input object, can we create data in the format that we expect later in the state machine or in subsequent invocations? I'm also good with just embedding the entire raw request inside the event object we pass along so that we can reference anything we want in it

Reply:
not sure I'm understanding the "embedding the entire raw request" piece here. The goal of this is to fix all of our PascalPropertyNames that we have from pydantic into camelPropertyNames that the CDK schema expects. We could (and probably should) go through and change them all to pascal case but I just did this in the interest of moving quickly.

Reply:
got it, I misunderstood the "why" of doing this. all good 👍